src/lib/notifications/backgroundActions.native.js

```diff
@@ -0,0 +1,46 @@
+import { useEffect } from 'react'
+import { Notifications } from 'react-native-notifications'
+import { t } from '@lingui/macro'
+import { LAST_CLAIM_NOTIFICATION } from '../constants/localStorage'
+import logger from '../logger/js-logger'
+
+const log = logger.child({ from: 'backgroundFetch' })
+
+export const CLAIM_NOTIFICATION = 'claim-notification'
+
+export const dailyClaimTime = new Date().setUTCHours(12)
+
+export const dailyClaimNotification = async (userStorage, goodWallet) => {
+  try {
+    const { entitlement: dailyUBI } = await goodWallet.getClaimScreenStatsFuse()
+
+    // We should notify once: only in first bg-fetch call after daily claim time
+    const lastClaimNotification = await userStorage.userProperties.get(LAST_CLAIM_NOTIFICATION)
```

Comment:
we dont use constants for user properties. just cameltext. ie lastClaimNotificaiton